sdk/testing/src/main/java/io/opentelemetry/sdk/testing/exporter/InMemoryMetricReader.java

```diff
@@ -52,13 +55,39 @@ public class InMemoryMetricReader implements MetricReader {
   private final DefaultAggregationSelector defaultAggregationSelector;
   private final AtomicBoolean isShutdown = new AtomicBoolean(false);
   private volatile MetricProducer metricProducer = MetricProducer.noop();
+  private final MemoryMode memoryMode;
+
+  /**
+   * Creates an {@link InMemoryMetricReaderBuilder} with defaults.
+   *
+   * @return a builder with always-cumulative {@link AggregationTemporalitySelector}, default {@link
+   *     DefaultAggregationSelector} and {@link MemoryMode#IMMUTABLE_DATA} {@link MemoryMode}
+   * @since 1.29.0
+   */
+  public static InMemoryMetricReaderBuilder builder() {
+    return new InMemoryMetricReaderBuilder();
+  }
 
   /** Returns a new {@link InMemoryMetricReader}. */
   public static InMemoryMetricReader create() {
     return new InMemoryMetricReader(
         AggregationTemporalitySelector.alwaysCumulative(), DefaultAggregationSelector.getDefault());
   }
 
+  /**
+   * Create an {@link InMemoryMetricReader} with an {@link AggregationTemporalitySelector}.
+   *
+   * @param aggregationTemporalitySelector The {@link AggregationTemporalitySelector} to be used in
+   *     the reader
+   * @return a new {@link InMemoryMetricReader} which uses {@code aggregationTemporalitySelector}
+   */
+  public static InMemoryMetricReader create(
+      AggregationTemporalitySelector aggregationTemporalitySelector) {
+
+    return new InMemoryMetricReader(
+        aggregationTemporalitySelector, DefaultAggregationSelector.getDefault());
+  }
```

Comment:
Can remove this new create method now that we have the builder.

Reply:
I removed. Now it has `InMemoryMetricReader` has same create method as before. We can't remove existing create method  those completely because this is a public API right?